src/test/ui/symbol-names/impl1.stderr

```diff
@@ -1,26 +0,0 @@
-error: symbol-name(_ZN5impl13foo3Foo3bar17he53b9bee7600ed8dE)
```

Comment:
I'm not sure why this file was removed – I suspect that `--bless` did it.

Reply:
The corresponding `legacy.stderr` and `v0.stderr` files were updated though. Maybe some kind of "vanilla"  test mode was skipped?!